File: Udemy/tarea en java/notas.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        float notas1, nota2, nota3, promedio;

        System.out.println("¿Cuál es la primera nota?");
        notas1 = scanner.nextFloat();

        System.out.println("¿Cuál es la segunda nota?");
        nota2 = scanner.nextFloat();

        System.out.println("¿Cuál es la tercera nota?");
        nota3 = scanner.nextFloat();

        promedio = (notas1 + nota2 + nota3) / 3;

        if (promedio == 5) {
            System.out.println("El estudiante es excelente: " + promedio);
        } else if (promedio >= 4 && promedio < 5) {
            System.out.println("El estudiante es sobresaliente: " + promedio);
        } else if (promedio >= 3 && promedio < 4) {
            System.out.println("El estudiante es aceptable: " + promedio);
        } else if (promedio >= 2 && promedio < 3) {
            System.out.println("El estudiante es insuficiente: " + promedio);
        } else if (promedio >= 1 && promedio < 2) {
            System.out.println("El estudiante ha reprobado: " + promedio);
        } else if (promedio >= 0 && promedio < 1) {
            System.out.println("El estudiante ha reprobado: " + promedio);
        } else {
            System.out.println("La nota es inválida: " + promedio);
        }

        scanner.close();
    }
}

File: Udemy/lenguje c/notas.c
// premedio de tres notas 
//#include <stdio.h>
 
 int main (){
    float notas1,nota2,nota3, promedio, resultado;

    printf(" cual es la primera nota \n ");
    scanf("%f", &notas1);

    printf("Cual es la segunda nota \n ");
    scanf("%f", &nota2);

    printf("Cual es la tercera nota \n ");
    scanf("%f", &nota3);

    promedio = (notas1 + nota2 + nota3)/3;

    if(promedio == 5){
        printf("el estudiante es excelente: %f ",promedio);
    }else if (promedio >= 4 && promedio <5){ 
        printf("el estudiante es sobresaliente: %f ",promedio);
    }else if (promedio >= 3 && promedio <4){
        printf("el estudiante es aceptable e: %f ",promedio);
    }else if (promedio >= 2 && promedio <3){
        printf("el estudiante es es insuficiente: %f ",promedio);
    }else if (promedio >= 1 && promedio <2){
        printf("el estudiante a reprobado: %f",promedio);    
    }else if(promedio >= 0 && promedio <1){
        printf("el estudiante a reprobado: %f",promedio);
    }else{
        printf("la nota es invalida : %f",promedio);
    }     


 }